src/routes/(app)/admin[[hash=admin_hash]]/product/product-schema.ts

```diff
@@ -64,7 +64,11 @@ export const productBaseSchema = () => ({
 		.array(
 			z.object({
 				name: z.string().trim(),
-				value: z.string().trim()
+				value: z.string().trim(),
+				price: z
+					.string()
+					.default('0')
+					.transform((val) => Number(val))
```

Comment:
It's here that you should use `parsePriceAmount`, like for `priceAmount` above.

Here you can do the same regex as `priceAmount`, and call `parsePriceAmount` after the validation (once you know the currency used)

Reply:
Also do a min(0) check